src/Playwright/Transport/Protocol/ExpectedTextValue.cs

```diff
@@ -0,0 +1,46 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Playwright.Transport.Protocol
+{
+    internal class ExpectedTextValue
+    {
+        [JsonPropertyName("string")]
+        public string String { get; set; }
+
+        [JsonPropertyName("regexSource")]
+        public string RegexSource { get; set; }
```

Comment:
Are default serializer settings used here? Then you are duplicating what your serializer already does for you. 

https://github.com/microsoft/playwright-dotnet/blob/8a4756e820dcb295ba0762a95a2b2a6f197dd415/src/Playwright/Helpers/JsonExtensions.cs#L90-L99


Docs with example:

https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-customize-properties?pivots=dotnet-5-0#use-camel-case-for-all-json-property-names

Reply:
cool, TIL.